Mathlib/Topology/Constructions.lean

```diff
@@ -39,39 +39,39 @@ open Topology TopologicalSpace Set Filter Function Classical
 
 universe u v
 
-variable {α : Type u} {β : Type v} {γ δ ε ζ : Type*}
+variable {X : Type u} {Y : Type v} {Z δ ε ζ : Type*}
 
 section Constructions
 
-instance instTopologicalSpaceSubtype {p : α → Prop} [t : TopologicalSpace α] :
+instance instTopologicalSpaceSubtype {p : X → Prop} [t : TopologicalSpace X] :
     TopologicalSpace (Subtype p) :=
   induced (↑) t
 
-instance {r : α → α → Prop} [t : TopologicalSpace α] : TopologicalSpace (Quot r) :=
+instance {r : X → X → Prop} [t : TopologicalSpace X] : TopologicalSpace (Quot r) :=
   coinduced (Quot.mk r) t
 
-instance instTopologicalSpaceQuotient {s : Setoid α} [t : TopologicalSpace α] :
+instance instTopologicalSpaceQuotient {s : Setoid X} [t : TopologicalSpace X] :
     TopologicalSpace (Quotient s) :=
   coinduced Quotient.mk' t
 
-instance instTopologicalSpaceProd [t₁ : TopologicalSpace α] [t₂ : TopologicalSpace β] :
-    TopologicalSpace (α × β) :=
+instance instTopologicalSpaceProd [t₁ : TopologicalSpace X] [t₂ : TopologicalSpace Y] :
+    TopologicalSpace (X × Y) :=
   induced Prod.fst t₁ ⊓ induced Prod.snd t₂
 
-instance instTopologicalSpaceSum [t₁ : TopologicalSpace α] [t₂ : TopologicalSpace β] :
-    TopologicalSpace (α ⊕ β) :=
+instance instTopologicalSpaceSum [t₁ : TopologicalSpace X] [t₂ : TopologicalSpace Y] :
+    TopologicalSpace (X ⊕ Y) :=
   coinduced Sum.inl t₁ ⊔ coinduced Sum.inr t₂
 
-instance instTopologicalSpaceSigma {β : α → Type v} [t₂ : ∀ a, TopologicalSpace (β a)] :
-    TopologicalSpace (Sigma β) :=
+instance instTopologicalSpaceSigma {X : X → Type v} [t₂ : ∀ a, TopologicalSpace (X a)] :
```

Comment:
It should be `X : α → Type*`, not `X : X → Type*`.

Reply:
Sure. I've changed to use \iota as indexing type (as many other files also do).